File: ming/0915/Unique Characters.java
public class Solution {
    /**
     * @param str: a string
     * @return: a boolean
     */
    public boolean isUnique(String str) {
        boolean[] unique = new boolean[256];
        for (int i = 0; i < str.length(); i++){
            if (unique[str.charAt(i)] == true){
                return false;
            }else{
                unique[str.charAt(i)] = true;
            }
        }
        return true;
    }
}

File: mu_wang/9_15/Unique Characters.py
class Solution:
    # @param s: a string
    # @return: a boolean
    def isUnique(self, str):
        # write your code here
        count = dict()
        for i in str:
            count[i] = count.get(i,0)+1
        for a in count:
            if count[a]>1:
                return False
        return True


    def isUnique_2(self, str):
        # write your code here
        for i in range(0,len(str)):
            for a in range(i+1,len(str)):
                if str[i]==str[a]:
                    return False
        return True